src/components/buttons/button-accessibility-image-alt.jsx

```diff
@@ -23,19 +23,22 @@ class ButtonAccessibilityImageAlt extends React.Component {
 	constructor(props) {
 		super(props);
 
-		const element = props.context.editor
+		const selection = props.context.editor
 			.get('nativeEditor')
-			.getSelection()
-			.getSelectedElement();
+			.getSelection();
 
-		const imageElement = element.findOne('img');
+		const element =
+			selection.getSelectedElement() || selection.getStartElement();
+
+		this._element = element;
+
+		const imageElement = this._element.findOne('img');
 
 		const imageAlt = imageElement
 			? imageElement.getAttribute('alt')
-			: element && element.getAttribute('alt');
+			: this._element && this._element.getAttribute('alt');
```

Comment:
`this._element &&` seems superflous here. If it wasn't truthy, line 35 would have thrown an error when you do `this._element.findOne('img')`.